### PR TITLE
Added <= and >= operators to expression rearrangement functionality.

### PR DESCRIPTION
**Pull request by ragavsachdeva**

### CODE REVIEW

**Comment by alxrsngrtn** [comment]
I'm grateful for the PR to land, since we're getting spammed with the following error: 

```
Error: Unsupported operator >=: cannot rearrange numerical expression.
    at Function.rearrangeNumericalExpression (refiner.ts:1229)
    at BinaryExpression.rearrange (refiner.ts:291)
    at Refinement.normalise (refiner.ts:145)
    at Function.isAtleastAsSpecificAs (refiner.ts:127)
    at Schema.isAtleastAsSpecificAs (schema.ts:173)
    at EntityType._isAtleastAsSpecificAs (type.ts:348)
    at EntityType.isAtleastAsSpecificAs (type.ts:198)
    at Function.getResolution (type-checker.ts:35)
    at Function.processTypeList (type-checker.ts:97)
    at Function.resolveEffectiveType (handle.ts:232)
```

**Comment by ragavsachdeva** [comment]
> I'm grateful for the PR to land, since we're getting spammed with the following error:
> 
> ```
> Error: Unsupported operator >=: cannot rearrange numerical expression.
>     at Function.rearrangeNumericalExpression (refiner.ts:1229)
>     at BinaryExpression.rearrange (refiner.ts:291)
>     at Refinement.normalise (refiner.ts:145)
>     at Function.isAtleastAsSpecificAs (refiner.ts:127)
>     at Schema.isAtleastAsSpecificAs (schema.ts:173)
>     at EntityType._isAtleastAsSpecificAs (type.ts:348)
>     at EntityType.isAtleastAsSpecificAs (type.ts:198)
>     at Function.getResolution (type-checker.ts:35)
>     at Function.processTypeList (type-checker.ts:97)
>     at Function.resolveEffectiveType (handle.ts:232)
> ```

@alxrsngrtn Sorry about that. I've pushed a quick patch https://github.com/PolymerLabs/arcs/pull/4578 while we work on this PR. Please let me know if you still encounter this error.

File: src/runtime/refiner.ts [hunk 1]
```diff
@@ -343,6 +348,28 @@ export class BinaryExpression extends RefinementExpression {
     }
   }
 
+  normaliseOperators(): RefinementExpression {
```
Comment (Cypher1) [inline src/runtime/refiner.ts]: Can you open a PR to change from AU+UK spelling to US for normali[sz]e?

Comment (ragavsachdeva) [reply src/runtime/refiner.ts]: Ah yes, I'll do that!

**Pull request merged**